### DISCUSSION TITLE
Come faccio a creare il registro per il mio istituto

### DISCUSSION THREAD

**Discussion by Giuppimack65** [iisgiua/giuaschool#674] [Richieste di aiuto] [answered]
-

**Comment by trinko** [comment]
Devi fare l'installazione su un servizio di hosting.
Vedi [la documentazione](https://iisgiua.github.io/giuaschool-docs/install-zip.html)

**Comment by Giuppimack65** [comment]
salve, purtroppo non ci riusciamo, non potrebbe aiutarci?
Cordialmente
G.M

Il giorno lun 25 mar 2024 alle ore 11:35 Antonello Dessì <
***@***.***> ha scritto:

**Comment by trinko** [comment, marked as answer]
La procedura per l'installazione usando un servizio di hosting come Altervista è descritta di seguito: se non riesci ad eseguire uno dei passi, indica nel dettaglio i problemi riscontrati. 

1. Se non l'hai già fatto, registrati su Altervista e scegli il nome del dominio.
1. Vai al pannello di controllo di Altervista, quindi clicca sul menu **RISORSE**;
   - cerca il riquadro PHP e clicca sul collegamento **"Modifica Impostazioni PHP"**;
   - a questo punto cerca l'impostazione della **versione PHP**, impostala su **"PHP8"** e clicca sul pulsante **SALVA**.
1. Scarica il [file zip dell'ultima versione](https://iisgiua.github.io/giuaschool-docs/latest-release.html) e decomprimilo sul tuo computer.
1. Usa un programma come FileZilla ([scaricalo da qui se non ce l'hai](https://filezilla-project.org/download.php?platform=win64) );
    FileZilla ti permette di caricare i file sul tuo spazio Altervista, se non conosci questo programma: [vedi il tutorial FileZilla](https://www.youtube.com/watch?v=8kOMTK3XOp8)
1. A questo punto devi connetterti con FileZilla al tuo spazio Altervista, utilizzando le credenziali FTP che ti hanno dato loro;
    - i file del tuo spazio Altervista appariranno nel riquadro **SITO REMOTO**;
    - imposta nel riquadro **SITO LOCALE** la cartella presente nel tuo computer con i file estratti dallo ZIP di giuaschool;
    - effettua il trasferimento di tutti i file **dal sito LOCALE al sito REMOTO** (richiederà un po' di tempo, sono tanti file);
    - controlla alla fine se ci sono stati errori: in tal caso ricarica solo i file indicati come errati.
    
1. quando tutto è stato caricato puoi eseguire la procedura di installazione andando all'indirizzo:
    **http://nome_sito_altervista/public/install/app.php**
